packages/eslint-plugin-query/src/__tests__/exhaustive-deps.test.ts

```diff
@@ -354,24 +354,6 @@ ruleTester.run('exhaustive-deps', rule, {
         }
       `,
     },
-    {
-      name: 'should ignore references of the queryClient',
```

Comment:
note: we don’t need an exception for closing over the `queryClient` anymore if we inject it into the `queryFn`. @Newbie012 would you agree?

Reply:
I guess so. I never had to deal with more than one query client in a single app. Is it a valid use case to reference a foreign query client inside queryFn?